nni/common/concrete_trace_utils/concrete_proxy.py

```diff
@@ -40,7 +40,7 @@
 class ConcreteProxy(Proxy):
     """
     `ConcreteProxy` is a wrapped proxy carried the real intermediate value.
-    We can use it to trace a more compatible model, and pass the branches.
+    We can use it to trace a more compatibal model, and pass the branches.
```

Comment:
compatible

Reply:
Apology. This spelling mistake has been corrected.